public/index.js

```diff
@@ -64,8 +64,12 @@ buttons.forEach((button) => {
       .then((response) => response.json())
       .then((data) => {
         if (resultElement) {
-          resultElement.innerHTML = `<p>${data.choices[0].message.content}</p>`;
```

Comment:
Deleted this as i've made the resultElement to include textcontext and imageUrl. @josueJURE